cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/BridgeConfiguration.java

```diff
@@ -19,22 +20,24 @@
 
     private String prefix;
     private boolean onlyProxyProtection = true;
-    private Collection<String> excludedOnlyProxyWalkableGroups = new ArrayList<>(), excludedGroups;
+    private Collection<String> excludedOnlyProxyWalkableGroups, excludedGroups;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5780263)